packages/local_auth/local_auth_android/android/build.gradle

```diff
@@ -29,7 +29,7 @@ android {
     compileSdk 34
 
     defaultConfig {
-        minSdkVersion 16
+        minSdkVersion flutter.minSdkVersion
```

Comment:
I think this needs to be a 19

Reply:
A lot of our existing examples use `flutter.minSdkVersion`; I thought we only wanted to use a number when a specific example needed something higher than the min (e.g., the example for a plugin that requires 21).

Reply:
Wait, this isn't the `/app/` file. Why is this line even here? I need to look at this again, I missed the path on this.

Reply:
Oh, it's not the example, it's the actual plugin. I really can't read this morning.

Now I'm confused as to what's causing the error in the example app though. This should probably be 19 and get a version bump, but the example app is already using `flutter.minSdkVersion`.